AUTHORS.csv

```diff
@@ -1,3 +1,6 @@
 Full Name, Surname First, Name Reference
 Ernest W. Durbin III, "Durbin, Ernest W., III", Durbin
-Inada Naoki, "Inada, Naoki", Inada
\ No newline at end of file
+Inada Naoki, "Inada, Naoki", Inada
+Guido van Rossum, "van Rossum, Guido (GvR)", GvR
+Just van Rossum, "van Rossum, Just (JvR)", JvR
+The Python core team and community, The Python core team and community, The Python core team and community
```

Comment:
Hah good catch!  If you need a short reference for this last entry, you could use `python-dev`.

Reply:
Thanks!